client/src/components/InputsModal/InputsModal.tsx

```diff
@@ -56,29 +56,51 @@ const InputsModal: FC<InputsModalProps> = ({
     }
     const inputs_with_values: TestInput[] = [];
     inputsMap.forEach((input_value, input_name) => {
-      inputs_with_values.push({ name: input_name, value: input_value });
+      inputs_with_values.push({ key: input_name, value: input_value });
     });
     createTestRun(runnableType, runnableId, inputs_with_values);
     hideModal();
   }
   const [inputsMap, setInputsMap] = React.useState<Map<string, string>>(new Map());
 
   const inputFields = inputs.map((requirement: TestInput, index: number) => {
-    return (
-      <ListItem key={`requirement${index}`}>
-        <TextField
-          id={`requirement${index}_input`}
-          fullWidth
-          label={requirement.name}
-          value={inputsMap.get(requirement.name) || ''}
-          onChange={(event) => {
-            const value = event.target.value;
-            inputsMap.set(requirement.name, value);
-            setInputsMap(new Map(inputsMap));
-          }}
-        />
-      </ListItem>
-    );
+    switch (requirement.type) {
+      case 'textarea':
+        return (
+          <ListItem key={`requirement${index}`}>
+            <TextField

```

Comment:
Is it possible to make this resizable? Four lines is better than a normal text input, but still seems pretty inadequate for something like a FHIR resource.

Reply:
Updated to be resizable.

Reply:
Ok, can its default size be increased? It's showing up initially as just a single line for me.
![Screen Shot 2021-07-01 at 11 59 06 AM](https://user-images.githubusercontent.com/15969967/124154729-c19f6b80-da63-11eb-926c-f9ed4164c3f5.png)

Reply:
Yeah, it now shows 4 lines initially.